versioned_docs/version-22.04/upgrade/upgrade-mariadb.md

```diff
@@ -60,6 +60,20 @@ You have to uninstall then reinstall MariaDB to upgrade between major versions (
     rpm --erase --nodeps --verbose MariaDB-server MariaDB-client MariaDB-shared MariaDB-compat MariaDB-common
     ```
 
+    If you have an error while performing this uninstallation step, please execute:
```

Comment:
```suggestion
    If you have an error while performing this uninstallation step because **MariaDB-shared** is not installed, please execute following command:
```